Spanner/src/Connection/Grpc.php

```diff
@@ -1580,4 +1601,46 @@ private function getLroResponseMapper($typeUrl)
 
         return null;
     }
+
+    private function getDirectedReadOptions($args)
+    {
+        $directedReadOptions = [];
+        if (!empty($args['includeReplicas'])) {
+            $includeReplicas = [
+                'auto_failover_disabled' => !($args['includeReplicas']['failover'] ?? true)
+            ];
```

Comment:
nit: is it required to take in `failover` and convert to `auto_failover_disabled` but not take in `auto_failover_disabled` directly?

Reply:
I felt the term is confusing to the user, so used an easy term!

Reply:
Changed this to `auto_failover_disabled `